We conclude our dialogue between back-of-the-envelope estimates and comparison
with the proteomic data by examining the final process in the central dogma --
translation. In doing so, we will begin with an estimate of the number of
ribosomes needed to replicate the cellular proteome. While the rate at which
ribosomes translate is well known to depend on the growth rate [\cite{dai2018},
a phenomenon we consider later in this work] we begin by making the
approximation that translation occurs at a modest rate of $\approx$ 15 amino
acids per second per ribosome (BNID: 100233). Under this approximation and our
previous estimate of 10$^{9}$ peptide bonds per cell at a growth rate of 0.5
hr$^{-1}$, we can easily arrive at an estimate of $\approx 10^4$ ribosomes
needed per cell to replicate the entire protein mass \FIG{central_dogma}(E) and
proves comparable to the experimental observations. While the ribosome is
responsible for the formation of peptide bonds, we do not diminish the
importance of charging tRNAs with their appropriate amino acid, a process with
occurs with remarkable fidelity. In \FIG{central_dogma}(F) we show our estimate
for the required number of tRNA synthetases, which show similar accord with the
experimental data and is discussed further in the Supplemental Section "Estimates across
Fundamental Biological Processes."

Having completed our circuit through key processes of cellular growth outlined
in \FIG{categories}, we can now take stock of our understanding of the observed
growth rate dependence and abundances of various protein complexes. We note
that, broadly speaking, these simple estimates have been reasonably successful
in quantitatively describing the observations in the proteomic data, and
importantly suggests that the proteome is tuned in composition and absolute
abundance to match the growth rate requirements without any one process
representing a singular bottleneck or rate limiting step in division. However,
in our effort to identify key limitations on growth, there are two notable
observations worthy of additional emphasis.

The first is a recurring theme throughout the estimates investigated here, which
is that any inherent biochemical rate limitation can be overcome by expressing
more proteins. We can view this as a parallelization of each biosynthesis task,
which helps explain why bacteria tend to increase their protein content and
cell size as growth rate increases \citep{ojkic2019}. The second, and
ultimately the most significant in defining the cellular growth rate, is that
the synthesis of ribosomal proteins presents a special case where
parallelization is \textit{not} possible and thereby imposes a limit on the
fastest possible growth rate. Each ribosome has $\approx$ 7500 amino acids
across all of its protein components which must be strung together as peptide
bonds through the action of another ribosome. Once again using a modest
elongation rate of $\approx$ 15 amino acids per second, we arrive at an estimate of
$\approx$ 500 seconds or $\approx$ 7 minutes to replicate a single ribosome.
This limit, as remarked upon by others \citep{dill2011}, serves as a hard
theoretical boundary for how quickly a bacterium like \textit{E. coli} can
replicate.

% As each ribosome would therefore need to copy itself, this 7 minute
% speed limit is independent of the number of ribosomes per cell
% [\FIG{protein_synthesis}(B)], yet assumes that the only proteins that need to be
% replicated for division to occur are ribosomal proteins, a regime
% not met in biological reality. This poses an optimization problem for the cell
% -- how are the translational demands of the entire proteome met without
% investing resources in the production of an excess of ribosomes?
%
% This question, more frequently presented as a question of optimal resource
% allocation, has been the target of an extensive dialogue between experiment and
% theory over the past decade. In a now seminal work, \cite{scott2010} present an
% elegant treatment of resource allocation through partitioning of the proteome
% into sectors -- one of which being ribosome-associated proteins whose relative
% size ultimately defines the total cellular growth rate. In more recent years,
% this view has been more thoroughly dissected experimentally
% \citep{klumpp2014,basan2015,dai2018, dai2016, erickson2017}. However, the
% quantitative description of these  observations is often couched in terms of
% phenomenological constants and effective parameters with the key observable
% features of expression often computed in relative, rather than absolute,
% abundances. Furthermore, these approaches often exclude or integrate away
% effects of cell size and chromosome content, which we have found through our
% estimates to have important connections to the observed cellular growth rate
% and proteomic content.

% In the closing sections of this work, we explore how ribosomal content, total
% protein abundance, and chromosomal replication are intertwined in their control
% over the cellular growth rate. To do so, we take a more careful view of ribosome
% abundance, increasing the sophistication of our analysis by exchanging our order-of-magnitude estimates for a minimal
% mathematical model of growth rate control. This is defined by parameters with
% tangible connections to the biological processes underlying cellular growth and
% protein synthesis. Using this model, we interrogate how the size of the ribosome
% pool and its corresponding translational capacity enable cells to maintain a
% balance  between the supply of amino acids via metabolism and catabolism and their
% consumption through the peptide bond formation required for growth.
%
