\section{Energy Production}
Cells consume and generate energy predominantly in the form of nucleoside
triphosphates (NTPs). The high-energy phosophodiester bonds of
(primarily) ATP power a variety of cellular processes that drive biological
systems away from thermodynamic equilibrium. We therefore turn to the synthesis
of ATP as another process that may limit growth, which will also require us
to consider the maintenance of the electrochemical proton gradient that powers
it.

Hydrolysis of the terminal phosphodiester bond of ATP into ADP (or
alternatively GTP and GDP) and an inorganic phosphate provides the
thermodynamic driving force in a wide array of biochemical reactions. One
such reaction is the formation of peptide bonds during translation, which
requires $\approx$ 2 ATPs for the charging of an amino acid to the tRNA and
$\approx$ 2 GTPs for the formation of each peptide bond. Assuming the ATP
costs associated with error correction and post-translational modifications
of proteins are negligible, we can make the approximation that each peptide
bond has a net cost of $\approx$ 4 ATP (BNID: 101442).
Formation of GTP from ATP is achieved via the action of nucleoside
diphosphate kinase, which catalyzes this reaction without an energy
investment \citep{lascu2000}. We therefore consider all NTP requirements of
the cell to be functionally equivalent to being exclusively ATP. In total,
the energetic costs of peptide bond formation consumes $\approx$ 80\% of the
cells ATP budget [BNID: 107782; 106158; 101637; 111918,
\cite{lynch2015,stouthamer1973}] and is primarily produced by the
F$_1$-F$_0$ ATP synthase -- a membrane-bound rotary motor which under ideal
conditions can yield $\approx$ 300 ATP per second [BNID: 114701;
\cite{weber2003}].

To estimate the total number of ATP equivalents consumed during a cell cycle, we
make the approximation that there are $\approx 3\times10^6$ proteins per cell
with an average protein length of $\approx$ 300 peptide bonds (BNID: 115702;
108986; 104877). With $\approx$ 4 ATP equivalents per peptide bond, we find that
the typical \textit{E. coli} cell consumes $\approx 5 \times 10^9$ ATP per cell
cycle on protein synthesis alone. Assuming that each ATP synthases operates at
its maximal speed, $\approx$ 3000 ATP synthases are needed to keep up with the
energy demands of the cell. This estimate is comparable with the experimental
observations, shown in \FIG{nutrient_cellwall_energy}(F). Since this estimate
assumes all ATP is synthesized via ATP synthase and neglects synthesis via
fermentative metabolism, this may explain why at the fastest growth rates
($\approx$ 2 hr$^{-1}$), our continuum estimate predicts more synthase than is
experimentally observed (data points below the gray line in
\FIG{nutrient_cellwall_energy}(F) at fast growth rates). Here, \textit{E. coli} enters
a type of overflow metabolism where non-respiratory routes for ATP synthesis
become more pronounced and provide the remaining ATP demand \citep{molenaar2009,
zhuang2011, szenk2017} .

In order to produce ATP, the F$_1$-F$_0$ ATP synthase itself must consume
energy. Rather than burning through its own product (and violating
thermodynamics), this intricate macromolecular machine has evolved to exploit
the electrochemical potential established across the inner membrane through
cellular respiration. This electrochemical gradient is manifest by the pumping
of protons into the intermembrane space via the electron transport chains as
they reduce NADH. In \textit{E. coli}, this potential difference is $\approx
-$200 mV (BNID: 102120). However, each rotation of an ATP synthase shuttles
$\approx$ 4 protons into the cytosol (BNID: 103390). With a few thousand ATP
synthases producing ATP at their maximal rate, the potential difference would be
rapidly abolished in a few milliseconds if it were not being actively
maintained. A recent work \citep{szenk2017} examined the respiratory capacity of
the \textit{E. coli} electron transport complexes using structural and
biochemical data, revealing that each electron transport chain rapidly pumps
protons into the intermembrane space at a rate of $\approx$ 1500 protons per
second (BIND: 114704; 114687). Using our estimate of the number of ATP synthases
required per cell, coupled with these recent measurements, we estimate that
$\approx 3000$ electron transport complexes would be necessary to facilitate the
$\approx 5 \times 10^6$ protons per second diet of the cellular ATP synthases.
This estimate is in agreement with the number of complexes identified in the
proteomic datasets [\FIG{nutrient_cellwall_energy}(G)]. This suggests
that every ATP synthase must be accompanied by $\approx$ 1 functional electron
transport chain.

\section{Limits on Biosynthesis within a Crowded Cell}
Our estimates thus far have focused on the biochemistry at the periphery of the
cell, with the processes of nutrient transport, cell envelope biogenesis, and
energy generation all requiring space to perform their biological functions. The
cell's surface area, however, does not scale as rapidly as cell size
\citep{harris2018} and there will be diminishing space available at the
periphery to support the proteomic requirements at faster growth rates. It is
therefore necessary to consider the consequences of a changing cell size and surface area to
volume ratio in our effort to identify limitations on growth. Here we use our
analysis of ATP production to better understand this constraint.

In our estimate of ATP production above we found that a cell demands about $5
\times 10^9$ ATP per cell cycle or $\approx 1 \times 10^6$ ATP/s. With a cell
volume of roughly 1 fL (BNID: 100004), this corresponds to about $2 \times
10^{10}$ ATP per fL of cell volume, in line with previous estimates
\citep{stouthamer1977, szenk2017}. In \FIG{energy_scaling}(A) we plot this ATP
demand as a function of the surface area to volume ratio in green, where we have
considered a range of cell shapes from spherical to rod-shaped with an aspect
ratio (length/width) equal to 4. In order to consider the maximum ATP that could
be produced, we consider the amount of ATP that can be generated by a membrane
filled with ATP synthase and electron transport complexes and a maximal
production rate of about 3 ATP / (nm$^2 \cdot$s) \citep{szenk2017}. This is
shown in blue in \FIG{energy_scaling}(A), which shows that at least for the
growth rates observed (right column in plot), the energy demand is roughly an
order of magnitude less. Interestingly, \cite{szenk2017} found that ATP
production by respiration is less efficient than by fermentation on a per
membrane area basis, due to the additional proteins of the electron transport
chain. This suggests that, even under anaerobic growth, cells will have
sufficient membrane space for ATP production.

Importantly, this analysis highlights that there will indeed be a maximum
attainable cell size due to the limited capacity to provide resources as the
cell increases in size. The maximum energy production in
\FIG{energy_scaling}(A), however, does represent a somewhat unachievable limit
since the inner membrane also includes other proteins like those we've
considered for nutrient transport and cell wall biogenesis. To better understand
the overall proteomic makeup of the inner membrane, we therefore used Gene
Ontology (GO) annotations \citep{ashburner2000, thegeneOntologyconsortium2018}
to identify all proteins embedded or peripheral to the inner membrane (GO term:
0005886). Those associated but not membrane-bound include proteins like MreB and
FtsZ that must nonetheless be considered as a vital component occupying space on
the membrane. Surprisingly, we find that the total protein mass per \textmu
m$^2$ is nearly constant across growth rates [\FIG{energy_scaling}(B)],
even though the volumetric demand for resources grows with cell size and growth
rate. Interestingly, when we consider the distribution of proteins grouped by
their Clusters of Orthologous Groups (COG) \citep{tatusov2000}, the relative
abundance of each category is nearly constant across growth rates
[\FIG{energy_scaling}(C)]. This suggests that no one process (energy production,
nutrient uptake, etc.) is dominating even at fast growth rates and in line with
our supposition that each of the processes we've considered so far are not
fundamentally limiting the maximum growth rate.

In contrast, when we apply such an analysis to cytosolic proteins
(GO term: 0005829), we observe a clear change in the proteomic composition
[\FIG{energy_scaling}(D, E)]. In particular, with increasing growth rates there
is a substantial increase in the relative protein mass associated with
'information storage and processing'. This category includes proteins such as
DNA polymerase, RNA polymerase, and ribosomes that are associated with the
processes of the central dogma, whose increase is predominantly at the expense
of 'metabolic' proteins as shown in \FIG{energy_scaling}(E). The notable
anticorrelation  provides a more extensive characterization of a trend that is
consistent with previous reports \citep{scott2010, hui2015, zhu2019}. In the
next section we therefore turn our attention to the processes of the central
dogma.

\begin{figure}
    \begin{fullwidth}
        \centering{
            \includegraphics[width=1\textwidth]{main_figs/fig3_energy_SV_scaling.pdf}
            \caption{\textbf{Influence of cell size and surface area to volume
            ratio on ATP production and inner membrane composition.}
        (A) Scaling of ATP demand and maximum ATP production through respiration as a
            function of surface area to volume ratio. Cell volumes of 0.5 fL to
            50 fL were considered, with the dashed (\texttt{- -}) line
            corresponding to a sphere and the dash-dot line (\texttt{-.})
            reflecting a rod-shaped bacterium like \textit{E. coli} with a
            typical aspect ratio (length / width) of 4 \citep{shi2018}. The ATP
            demand is calculated as $10^6$ ATP/(\textmu m$^3$ s), while the
            maximum ATP production rate is taken to be 3 ATP / (nm$^2 \cdot$s)
            \citep{szenk2017}, with calculations of \textit{E. coli} volume and
            surface area detailed in Supplemental Section "Estimation of Cell Size
            and Surface Area". In this calculation, 50\% of the bacterial inner
            membrane is assumed to be protein, with the remainder lipid.
        (B) Total protein mass per \textmu m$^2$ calculated for proteins with
            inner membrane annotation (GO term: 0005886).
        (C) Relative protein abundances are grouped by
            their COG annotations ('metabolic', 'cellular processes and
            signaling', 'information storage and processing', and 'poorly
            characterized or not annotated') for the data from
            \cite{schmidt2016}. Metabolic proteins are further separated into
            respiration (F$_1$-F$_0$ ATP synthase, NADH dehydrogenase I,
            succinate:quinone oxidoreductase, cytochrome bo$_3$ ubiquinol
            oxidase, cytochrome bd-I ubiquinol oxidase) and carbohydrate
            transport (GO term: GO:0008643). Note that the elongation factor
            EF-Tu can also associate with the inner membrane, but was excluded
            in this analysis due to its high relative abundance (roughly
            identical to the summed protein shown in part (B)).
        (D) Relative cytosolic protein abundances (GO term: 0005886), grouped by their COG annotations, are
            plotted as a function of growth rate.
        (E) The relative cytosolic protein
            abundances (GO term: 0005886) associated with the 'information
            storage and processing' and 'metabolic' COG categories are plotted
            against each other and highlight the larger mass fraction devoted to
            'information storage and processing' at faster growth rates.
            }\label{fig:energy_scaling}
            }
                \end{fullwidth}
\end{figure}
