\section{Additional Analysis on Processes of Nutrient Transport}
\label{sec:SI_nutr_trans}

\subsection{Nitrogen Transport}
We must first address which elemental sources must require active
transport, meaning that the cell cannot acquire appreciable amounts simply
via diffusion across the membrane. The permeability of the lipid membrane to
a large number of solutes has been extensively characterized over the past
century. Large, polar molecular species (such as various sugar molecules,
sulfate, and phosphate) have low permeabilities while small, non-polar
compounds (such as oxygen, carbon dioxide, and ammonia) can readily diffuse
across the membrane. Ammonia, a primary source of nitrogen in typical
laboratory conditions, has a permeability on par with water ($\sim 10^5$
nm/s, BNID:110824). In nitrogen-poor conditions,
\textit{E. coli} expresses a transporter (AmtB) which appears to aid in
nitrogen assimilation, though the mechanism and kinetic details of transport
are still a matter of debate \citep{heeswijk2013a, khademi2004}. Beyond
ammonia, another plentiful source of nitrogen come in the form of glutamate,
which has its own complex metabolism and scavenging pathways. However,
nitrogen is plentiful in the growth conditions examined in this work,
permitting us to neglect nitrogen transport as a potential rate limiting
process in cell division in typical experimental conditions.


\subsection{Phosphorus and Sulfur Transport}
We now turn our attention towards other essential elements, namely phosphorus and
sulfur. Phosphorus is critical to the cellular energy economy in the form of
high-energy phosphodiester bonds making up DNA, RNA, and the NTP energy pool as
well as playing a critical role in the post-translational modification of
proteins and defining the polar-heads of lipids. In total, phosphorus
makes up $\approx$3\% of the cellular dry mass which in typical experimental conditions is in the form of inorganic phosphate. The cell membrane
has remarkably low permeability to this highly-charged and critical molecule,
therefore requiring the expression of active transport systems. In \textit{E. coli}, the proton
electrochemical gradient across the inner membrane is leveraged to transport
inorganic phosphate into the cell \citep{rosenberg1977}.
Proton-solute symporters are widespread in \textit{E. coli} \citep{ramos1977,
booth1979} and can have rapid transport rates of 50 to 100 molecules per second for
sugars and other solutes (BNID: 103159; 111777). As a more
extreme example, the proton transporters in the F$_1$-F$_0$ ATP synthase, which
use the proton electrochemical gradient for rotational motion, can shuttle
protons across the membrane at a rate of $\approx$ 1000 per second (BNID:
104890; 103390). In \textit{E.
coli} the PitA phosphate transport system has been shown to be very tightly coupled
with the proton electrochemical gradient with a 1:1 proton:phosphate
stoichiometric ratio \citep{harris2001, feist2007}. Taking the geometric mean of
the aforementioned estimates gives a plausible rate of phosphate transport on
the order of 300  per second. Illustrated in
\FIG{phospho_sulfo_tport}(A), we can estimate that $\approx$ 200
phosphate transporters are necessary to maintain an $\approx$ 3\% dry mass with
a 5000 s division time. This estimate is consistent with observation when we examine the
observed copy numbers of PitA in proteomic data sets (plot in
\FIG{phospho_sulfo_tport}(A)). While our estimate is very much in line with the
observed numbers, we emphasize that this is likely a slight overestimate of the
number of transporters needed as there are other phosphorous scavenging systems,
such as the ATP-dependent phosphate transporter Pst system which we have neglected.

Satisfied that there are a sufficient number of phosphate transporters
present in the cell, we now turn to sulfur transport as another potentially rate
limiting process. Similar to phosphate, sulfate is  highly-charged
and not particularly membrane permeable, requiring active
transport. While there exists a H+/sulfate symporter in \textit{E.
coli}, it is in relatively low abundance and is not well characterized
\citep{zhang2014}. Sulfate is predominantly acquired via the ATP-dependent ABC
transporter CysUWA system which also plays an important role in selenium
transport \citep{sekowska2000, sirko1995}. While specific kinetic details of
this transport system are not readily available, generic ATP transport
systems in prokaryotes transport on the order of 1 to 10 molecules per second
(BNID: 109035). Combining this generic
transport rate, measurement of sulfur comprising 1\% of dry mass, and a 5000
second division time yields an estimate of $\approx$ 1000 CysUWA
complexes per cell (\FIG{phospho_sulfo_tport}(B)). Once again, this estimate
is in notable agreement with proteomic data sets, suggesting that there are
sufficient transporters present to acquire the necessary sulfur. In a similar
spirit of our estimate of phosphorus transport, we emphasize that this is
likely an overestimate of the number of necessary transporters as we have
neglected other sulfur scavenging systems that are in lower abundance.


\section{Analysis on Processes of Cell Envelope Biogenesis}
\label{sec:SI_envelope}
% \section{Synthesis of the Cell Envelope}
The subjects of our estimates thus far have been localized to the periphery of
the cell, embedded within the hydrophobic lipid bilayer of the inner membrane.
As outlined in \FIG{energy_scaling}, cells could in principle increase the
expression of the membrane-bound ATP synthases and electron transport chains to
support a larger energy budget across a wide range of cell volumes and membrane
surface areas. This ability, however, is contingent on the ability of the cell
to expand the surface area of the cell by synthesizing new lipids and
peptidoglycan for the cell wall. In this next class of estimates, we will
turn our focus to these processes and consider the copy numbers of the relevant
enzymes.

\subsection{Lipid Synthesis}
The cell envelopes of gram negative bacteria (such as \textit{E. coli}) are
composed of inner and outer phospholipid bilayer membranes separated by a
$\approx 10$ nm periplasmic space (BNID: 100016, \cite{milo2010}). As mentioned
in our discussion of the surface area to volume constraints on energy
production, \textit{E. coli} is a rod-shaped bacterium with a 4:1
length-to-width aspect ratio. At modest growth rates, such as our stopwatch of
5000 s, the total cell surface area is $\approx$ 5 \textmu m$^2$ (BNID: 101792,
\cite{milo2010}). As there are two membranes, each of which composed of two
lipid leaflets, the total membrane area is $\approx 20$\textmu m$^2$, a
remarkable value compared to the $\approx$ 2\textmu m length of the cell.

While this represents the total area of the membrane, this does not mean that it
is composed entirely of lipid molecules. Rather, the dense packing of the
membrane with proteins means that only $\approx$ 40 \% of the membrane area is
occupied by lipids (BNID: 100078). Using a rule-of-thumb of 0.5
nm$^2$ as the surface area of the typical lipid (BNID: 106993),
we arrive at an estimate of $\sim$ 2 $\times$ 10$^7$ lipids per cell, an
estimate in close agreement with experimental measurements (BNID: 100071,
102996).

The membranes of \textit{E. coli} are composed of a variety of different lipids,
each of which are unique in their structures and biosynthetic pathways
\citep{sohlenkamp2016}. With such diversity in biosynthesis, it becomes
difficult to identify which step(s) may be the rate-limiting, an objective further complicated by the sparsity of \textit{in vivo} kinetic data.
Recently, a combination of stochastic kinetic modeling \citep{ruppe2018} and
\textit{in vitro} kinetic measurements \citep{ranganathan2012, yu2011} have
revealed remarkably slow steps in the fatty acid synthesis pathways which may
serve as the rate limiting reactions. One such step is the removal of hydroxyl
groups from the fatty-acid chain by ACP dehydratase that leads to the formation of
carbon-carbon double bonds. This reaction, catalyzed by proteins FabZ and
FabA in \textit{E. coli} \citep{yu2011}, have been estimated to have kinetic
turnover rates of $\approx$ 1 dehydration per second per enzyme
\citep{ruppe2018}. Combined with this rate, our previous estimates for the
number of lipids to be formed, and a 5000 second division yields an estimate
that the cell requires $\approx$ 4000 ACP dehydratases. This is in
reasonable agreement with the experimentally observed copy numbers of FabZ and
FabA (\FIG{cell_envelope}(A)). Furthermore, we can extend this estimate to
account for the change in membrane surface area as a function of the growth rate
(grey line in \FIG{cell_envelope}(A)), which captures the observed growth rate
dependent expression of these two enzymes.

Despite the slow catalytic rate of FabZ and FabA, we argue that the generation
of fatty acids is not a bottleneck in cell division and is not the key process
responsible for setting the bacterial growth rate. Experimental evidence has
shown that the rate of fatty-acid synthesis can be drastically increased
\textit{in vitro} by increasing the concentration of FabZ \cite{yu2011}.
Stochastic simulations of the complete fatty acid synthesis pathway of
\textit{E. coli} further supports this experimental observation
\cite{ruppe2018}. Thus, if this step was the determining factor in cell
division, increasing growth rate could be as simple as increasing the number of
ACP dehydratases per cell. With a proteome size of $\approx$ 3$\times$10$^6$
proteins, a hypothetical increase in expression from 4000 to 40,000 ACP
dehydratases would result in a $\approx$ 1\% increase in the size of the proteome. As
many other  proteins are in much larger abundance than 4000 per cell (as we will
see in the coming sections), it is unlikely that expression of ACP dehydratases
couldn't be increased to facilitate faster growth.


\subsection{Peptidoglycan Synthesis}
While variation in cell size can vary substantially across growth conditions,
bacterial cells demonstrate exquisite control over their cell shape. This is
primarily due to the cell wall, a stiff meshwork of polymerized discaccharides
interspersed with short peptide crosslinks termed the peptidoglycan. The cell
wall is also a vital structural component that counteracts turgor pressure. In
\textit{E. coli}, this enormous peptidoglycan molecule is a few nanometers thick
and resides within the periplasmic space between the inner and outer membrane.
The formation of the peptidoglycan is an intricate process, involving the
bacterial actin homolog MreB \citep{shi2018} along with a variety of
membrane-bound and periplasmic enzymes \citep{morgenstein2015}. The coordinated
action of these components result in a highly-robust polymerized meshwork that
maintains cell shape even in the face of large-scale perturbations and can
restore rod-shaped morphology even after digestion of the peptidoglycan
\citep{harris2018,shi2018}.

In glucose-supported steady-state growth, the peptidoglycan alone comprises
$\approx$ 3\% of the cellular dry mass (BNID: 101936), making
it the most massive molecule in \textit{E. coli}. The polymerized unit of the
peptidoglycan is a N-acetylglucosamine and N-acetylmuramic acid disaccharide,
of which the former  is functionalized with a short pentapeptide. With a mass of
$\approx$ 1000 Da, this unit, which we refer to as a murein monomer, is
polymerized to form long strands in the periplasm which are then attached to
each other via their peptide linkers. Using the aforementioned measurement that
$\approx$ 3\% of the dry mass is peptidoglycan, it can be estimated that the
peptidoglycan is composed of $\sim$ 5$\times$10$^6$ murein monomers.

During growth, peptidoglycan is constantly being broken down to allow insertion
of new murein monomers and cellular expansion. In order to maintain structural
integrity these monomers must be crosslinked into the expanding cell wall,
potentially limiting how quickly new material can be added and we consider this
process as a possible rate-limiting step. In principle,  each one of these
murein monomers can be crosslinked to another glycan strand via the
pentapeptide.  In some species, such as in gram-positive bacterium
\textit{Staphylococcus aureus}, the extent of crosslinking can be large with $>$
90\% of pentapeptides forming a connection between glycan strands. In \textit{E.
coli}, however, a much smaller proportion ($\approx$ 20\%) of the peptides are
crosslinked, resulting in a weaker and more porous cell wall \cite{vollmer2008a,
rogers1980}.  The formation of these crosslinks primarily occur during the
polymerization of the murein monomers and is facilitated by a family of enzymes
called transpeptidases. In \textit{E. coli}, there are four primary
transpeptidases that are involved in lateral and longitudinal extension of the
peptidoglycan. These transpeptidases have only recently been quantitatively
characterized \textit{in vivo} via liquid chromatography mass spectrometry
\citep{catherwood2020}, which revealed a kinetic turnover rate of $\approx
2$ crosslinking reactions formed per second per enzyme.

Pulling these measurements together permits us to make an estimate that on the
order of $\approx$ 100 transpeptidases are needed for complete maturation of the
peptidoglycan, given a division time of $\approx$ 5000 seconds, a value that is
closely aligned with the experimental observations (\FIG{cell_envelope}(B)).
Expanding this estimate to account for the changing volume of the peptidoglycan
as a function of growth rates (grey line in \FIG{cell_envelope}(B)) also
qualitatively captures the observed dependence in the data, though systematic
disagreements between the different data sets makes the comparison more
difficult.

Much as in the case of fatty acid synthesis, we find it unlikely that the
formation of peptidoglycan is a process which defines the rate of bacterial cell division.
The estimate we have presented considered only the transpeptidase enzymes that
are involved lateral and longitudinal elongation of the peptidoglycan (proteins
MrdA, MrdB, MrcA, and MrcB). This neglects the presence of other transpeptidases
that are present in the periplasm and also involved in remodeling and maturation
of the peptidoglycan. It is therefore possible that if this was setting the
speed limit for cell division, the simple expression of more
transpeptidases may be sufficient to maintain the structural integrity of the
cell wall.


\section{Additional Analysis on Processes of the Central Dogma}
\label{sec:SI_central_dogma}

\subsubsection{dNTP synthesis}
We begin our exploration DNA replication by examining the
production of the deoxyribonucleotide triphosphates (dNTPs). The four major
dNTPs (dATP, dTTP, dCTP, and dGTP) are synthesized \textit{de novo} in separate
pathways, requiring different building blocks. However, a critical step present
in all dNTP synthesis pathways is the conversion from ribonucleotide to
deoxyribonucleotide via the removal of the 3' hydroxyl group of the ribose ring
\citep{rudd2016}. This reaction is mediated by a class of enzymes termed
ribonucleotide reductases, of which \textit{E. coli} possesses two aerobically
active complexes (termed I and II) and a single anaerobically active enzyme. Due
to their peculiar formation of a radical intermediate, these enzymes have
received much biochemical, kinetic, and structural characterization.  One such
work \citep{ge2003} performed a detailed \textit{in vitro} measurement of the
steady-state kinetic rates of these complexes, revealing a turnover rate of
$\approx$ 10 dNTP per second.

Since this reaction is central to the synthesis of all dNTPs, it is reasonable
to consider the abundance of these complexes is a measure of the total dNTP
production in \textit{E. coli}. Illustrated schematically in \FIG{DNA_synthesis}
(A), we consider the fact that to replicate the cell's genome, on the order of
$\approx 10^7$ dNTPs must be synthesized. Assuming a production rate of 10 per
second per ribonucleotide reductase complex and a cell division time of 5000
seconds, we arrive at an estimate of $\approx$ 200 complexes needed per cell. As
shown in the bottom panel of \FIG{DNA_synthesis} (A), this estimate agrees with
the experimental measurements of these complexes abundances within $\approx 1/2$
an order of magnitude. Extension of this estimate across a continuum of growth
rate, including the fact that multiple chromosomes can be replicated at a given
time, is shown as a grey transparent line in \FIG{DNA_synthesis}(A). Similarly
to out point estimate, this refinement agrees well with the data, accurately
describing both the magnitude of the complex abundance and the dependence on
growth rate.

Recent work has revealed that during replication, the ribonucleotide reductase
complexes coalesce to form discrete foci colocalized with the DNA replisome
complex \citep{sanchez-romero2011}. This is particularly pronounced in
conditions where growth is slow, indicating that spatial organization and
regulation of the activity of the complexes plays an important role.


\subsubsection{mRNA}
To form a functional protein, all protein coding genes must first be
transcribed from DNA to form an mRNA molecule. While each protein requires an
mRNA blueprint, many copies of the protein can be synthesized from a single
mRNA. Factors such as strength of the ribosomal binding site, mRNA stability,
and rare codon usage frequency dictate the number of proteins that can be
made from a single mRNA, with yields ranging from 10$^1$ to 10$^4$ (BNID: 104186; 100196;
106254). Computing the geometric mean of this range yields
$\approx$ 1000 proteins synthesized per mRNA, a value that agrees with
experimental measurements of the number of proteins per cell ($\approx 3
\times 10^6$, BNID: 100088) and total number of mRNA per
cell ($\approx 3 \times 10^3$, BNID:100064).

This estimation captures the \textit{steady-state} mRNA copy number, meaning
that at any given time, there will exist approximately 3000 unique mRNA
molecules. To determine the \textit{total} number of mRNA that need to be
synthesized over the cell's lifetime, we must consider degradation of the mRNA.
In most bacteria, mRNAs are rather unstable with life times on the order of
several minutes (BNID: 104324; 106253; 111927; 111998). For
convenience, we assume that the typical mRNA in our cell of interest has a
typical lifetime of $\approx$ 300 seconds. Using this value, we can determine
the total mRNA production rate to maintain a steady-state copy number of 3000
mRNA per cell. While we direct the reader to the appendix for a more detailed
discussion of mRNA transcriptional dynamics, we state here that the total mRNA
production rate must be on the order of $\approx$ 15 mRNA per second. In
\textit{E. coli}, the average protein is $\approx$ 300 amino acids in length
(BNID: 108986), meaning that the corresponding mRNA is
$\approx$ 900 nucleotides which we will further approximate as $\approx$ 1000
nucleotides to account for the non-protein coding regions on the 5' and
3' ends. This means that the cell must have enough RNA polymerase molecules
about to sustain a transcription rate of $\approx 1.5 \times 10^4$ nucleotides
per second. Knowing that a single RNA polymerase polymerizes RNA at a clip of 40
nucleotides per second, we arrive at a comfortable estimate of $\approx$ 250 RNA
polymerase complexes needed to satisfy the mRNA demands of the cell. It is worth
noting that this number is approximately half of that required to synthesize
enough rRNA, as we saw in the previous section. We find this to be a striking
result as these 250 RNA polymerase molecules are responsible for the
transcription of the $\approx$ 4000 protein coding genes that are not ribosome
associated.

\subsubsection{tRNA}
The final class of RNA molecules worthy of quantitative consideration are the
tRNAs that are used during translation to map codon sequence on mRNA to specific amino acids.
Unlike mRNA or rRNA, each individual tRNA is remarkably short, ranging
from 70 to 95 nucleotides each (BNID: 109645; 102340). What
they lack in length, they make up for in abundance, with reported values ranging from
$\approx$5$\times$10$^4$ (BNID: 105280) to
$\approx$5$\times$10$^5$ (BNID: 108611). To test tRNA synthesis as a possible
growth-rate limiting stage, we will err towards a higher abundance of $\approx$
5$\times$10$^5$ per cell. Combining the abundance and tRNA length measurements,
we make the estimate that $\approx 5 \times 10^7$ nucleotides are sequestered in tRNA per cell.
Unlike mRNA, tRNA is remarkably stable with typical lifetimes \textit{in vivo}
on the order of $\approx$ 48 hours \citep{abelson1974,svenningsen2017} -- well
beyond the timescale of division. Once again using our rule-of-thumb for the
rate of transcription to be 40 nucleotides per second and assuming a division
time of $\approx$ 5000 seconds, we arrive at an estimate of $\approx$ 200 RNA
polymerases to synthesize enough tRNA. This requirement pales in comparison to
the number of polymerases needed to generate the rRNA and mRNA pools and can be
neglected as a significant transcriptional burden.



\subsubsection{RNA Polymerase and $\sigma$-factor Abundance}
These estimates, summarized in \FIG{RNA_synthesis} (A), reveal that synthesis of
rRNA  and mRNA are the dominant RNA species synthesized by RNA polymerase,
suggesting the need for $\approx$ 1000 RNA polymerases per cell. As is revealed
in \FIG{RNA_synthesis} (B), this estimate is about an order of magnitude below
the observed number of RNA polymerase complexes per cell ($\approx$ 5000 -
7000). The difference between the estimated number of RNA polymerase needed for
transcription and and
these observations are consistent with recent literature revealing that
$\approx$ 80 \% of RNA polymerases in \textit{E. coli} are not transcriptionally
active \citep{patrick2015}. Our estimate ignores the possibility that some
fraction is only nonspecifically bound to DNA, as well as the obstacles that RNA
polymerase and DNA polymerase present for each other at they move along the DNA
\citep{finkelstein2013}.

In addition, it is also vital to consider the role of $\sigma$-factors which
help RNA polymerase identify and bind to transcriptional start sites
\citep{browning2016}. Here we consider $\sigma^{70}$ (RpoD) which is the
dominant "general-purpose" $\sigma$-factor in \textit{E. coli}. While initially
thought of as being solely involved in transcriptional initiation, the past two
decades of single-molecule work has revealed a more multipurpose role for
$\sigma^{70}$ including facilitating transcriptional elongation
\citep{kapanidis2005, goldman2015, perdue2011,mooney2003,mooney2005}.
\FIG{RNA_synthesis} (B) is suggestive of such a role as the number of
$\sigma^{70}$ proteins per cell is in close agreement with our estimate of the
number of transcriptional complexes needed.

These estimates provide insight as to the observed magnitude of both RNA
polymerase and the $\sigma$-70 factor. As we have done in the previous sections,
and described in Appendix \nameref{sec:SI_continuum_est}, we can generalize these estimates
across a wide range of growth rates (grey line in \FIG{RNA_synthesis}(B). While
there remains some disagreement in the magnitude of the copy number, this
estimate appears to very adequately describe the growth rate dependence of these
complexes. Furthermore, these findings illustrate that transcription
cannot be the rate limiting step in bacterial division. \FIG{RNA_synthesis} (A)
reveals that the availability of RNA polymerase is not a limiting factor for
cell division as the cell always has an apparent $\sim$ 10-fold excess than needed.
Furthermore, if more transcriptional activity was needed to satisfy the cellular
requirements, more $\sigma^{70}$-factors could be expressed to utilize a larger
fraction of the RNA polymerase pool.



\subsubsection{tRNA Synthetases}
We begin by first estimating the number of tRNA synthetases in \textit{E.
coli} needed to convert free amino-acids to polypeptide chains. Again using
an estimate of $\approx$ 3$\times$10$^6$ proteins per cell at a 5000 s
division time (BNID: 115702) and a typical protein length of $\approx$ 300
amino acids (BNID: 100017), we can estimate that a total of $\approx$ 10$^9$
amino acids are stitched together by peptide bonds.

How many tRNAs are needed to facilitate this remarkable number of amino acid
delivery events to the translating ribosomes? It is important to note that tRNAs
are recycled after they've passed through the ribosome and can be recharged with
a new amino acid, ready for another round of peptide bond formation. While some
\textit{in vitro} data exists on  the turnover of tRNA in \textit{E. coli} for
different  amino acids, we can make a reasonable estimate by comparing the
number of amino acids to be  polymerized to cell division time. Using our
stopwatch of 5000 s and 10$^9$ amino acids, we arrive at a requirement of
$\approx$ 2 $\times$ 10$^5$ tRNA molecules to be consumed by the ribosome per
second.

There are many processes which go into synthesizing a tRNA and ligating it
with the appropriate amino acids. As we discussed previously, there appear to
be more than enough RNA polymerases per cell to synthesize the needed pool of
tRNAs. Without considering the many ways in which amino acids can be
scavenged or synthesized \textit{de novo}, we can explore ligation the as a
potential rate limiting step. The enzymes which link the correct amino acid
to the tRNA, known as tRNA synthetases or tRNA ligases, are incredible in
their proofreading of substrates with the incorrect amino acid being ligated
once out of every $10^4$ to $10^5$ events (BNID: 103469).
This is due in part to the consumption of energy as well as a multi-step
pathway to ligation. While the rate at which tRNA is ligated is highly
dependent on the identity of the amino acid, it is reasonable to state that
the typical tRNA synthetase has charging rate of $\approx$ 20 AA per tRNA
synthetase per second (BNID: 105279).

We can make an assumption that amino-acyl tRNAs are in steady-state where they
are produced at the same rate they are consumed, meaning that $2 \times 10^5$
tRNAs must be charged per second. Combining these estimates together, as shown schematically
in \FIG{protein_synthesis}(A), yields an estimate of $\sim$ 10$^4$ tRNA
synthetases per cell with a division time of 5000 s. This point estimate is in
very close agreement with the observed number of synthetases (the sum of all 20
tRNA synthetases in \textit{E. coli}). This estimation strategy seems to
adequately describe the observed growth rate dependence of the tRNA synthetase copy
number (shown as the grey line in \FIG{protein_synthesis}(B)), suggesting that
the copy number scales with the cell volume.

In total, the estimated and observed $\sim$ 10$^4$ tRNA synthetases occupy
only a meager fraction of the total cell proteome, around 0.5\% by abundance. It
is reasonable to assume that if tRNA charging was a rate limiting process, cells
would be able to increase their growth rate by devoting more cellular resources
to making more tRNA synthases. As the synthesis of tRNAs and the corresponding
charging can be highly parallelized, we can argue that tRNA charging is not a
rate limiting step in cell division, at least for the growth conditions explored
in this work.
