\section{Discussion}
Continued experimental and technological improvements have led to a treasure
trove of quantitative biological data \citep{hui2015, schmidt2016, si2017,
gallagher2020, peebo2015, valgepea2013}, and an ever advancing molecular view
and mechanistic understanding of the constituents that support bacterial
growth \citep{taheriaraghi2015, morgenstein2015, si2019, karr2012,
kostinski2020}. In this work we have compiled what we believe to be the
state-of-the-art knowledge on proteomic copy number across a broad range of
growth conditions in \textit{E. coli}. We have made this data accessible
through a \href{https://github.com/RPGroup-PBoC/growth_limits}{GitHub
repository}, and an interactive figure that allows exploration of specific
protein and protein complex copy numbers. Through a series of
order-of-magnitude estimates that traverse key steps in the bacterial cell
cycle, this proteomic data has been a resource to guide our understanding of
two key questions: what biological processes limit the absolute speed limit
of bacterial growth, and how do cells alter their molecular constituents as a
function of changes in growth rate or nutrient availability? While not
exhaustive, our series of estimates provide insight on the scales of
macromolecular complex abundance across four classes of cellular processes --
the transport of nutrients, the production of energy, the synthesis of the
membrane and cell wall, and the numerous steps of the central dogma.

In general, the copy numbers of the complexes involved in these processes were
reasonable agreement with our order-of-magnitude estimates. Since many of these
estimates represent soft lower-bound quantities, this suggests that cells do not
express proteins grossly in excess of what is needed for a particular growth
rate. Several exceptions, however, also highlight the dichotomy between a
proteome that appears to "optimize" expression according to growth rate and one
that must be able to quickly adapt to environments of different nutritional
quality. Take, for example, the expression of carbon transporters. Shown in
\FIG{carbon_tport}(B), we find that cells always express a similar number of
glucose transporters irrespective of growth condition. At the same time, it is
interesting to note that many of the alternative carbon transporters are still
expressed in low but non-zero numbers ($\approx$ 10-100 copies per cell) across
growth conditions. This may relate to the regulatory configuration for many of
these operons, which require the presence of a metabolite signal in order for
alternative carbon utilization operons to be induced \citep{monod1949,
laxhuber2020}. Furthermore, upon induction, these transporters are expressed and
present in abundances in close agreement with a simple estimate.

Of the processes illustrated in \FIG{categories}, we arrive at a
ribosome-centric view of cellular growth rate control. This is in some sense
unsurprising given the long-held observation that \textit{E. coli} and many
other organisms vary their ribosomal abundance as a function of growth
conditions and growth rate \cite{scott2010, metzlraz2017}. However, through our
dialogue with the proteomic data, two additional key points emerge. The first
relates to our question of what process sets the absolute speed limit of
bacterial growth. While a cell can parallelize many of its processes simply by
increasing the abundance of specific proteins or firing multiple rounds of DNA
replication, this is not so for synthesis of ribosomes
(\FIG{ribosome_limit}(A)). The translation time for each ribosome [$\approx$ 6
min, \cite{dill2011}] places an inherent limit on the growth rate that can only
be surpassed if the cell were to increase their polypeptide elongation rate, or
if they could reduce the total protein and rRNA mass of the ribosome. The second
point relates to the long-observed correlations between growth rate and cell
size \citep{schaechter1958, si2017}, and between growth rate and ribosomal mass
fraction. While both trends have sparked tremendous curiosity and driven
substantial amounts of research in their own regards, these relationships are
themselves intertwined. In particular, it is the need for cells to increase
their absolute number of ribosomes under conditions of rapid growth that require
cells to also grow in size. Further experiments are needed to test the validity
of this hypothesis. In particular, we believe that the change in growth rate in
response to translation-inhibitory drugs (such as chloramphenicol) could be
quantitatively predicted, given one had precision measurement of the relevant
parameters, including the fraction of actively translating ribosomes $f_a$ and
changes in the metabolic capacity of the cell (i.e. the parameter $r_{AA}$ in
our minimal model) for a particular growth condition.

While the generation of new ribosomes plays a dominant role in growth rate
control, there exist other physical limits to the function of cellular
processes. One of the key motivations for considering energy production was the
physical constraints on total volume and surfance area as cells vary their size
\citep{harris2016, ojkic2019}. While \textit{E. coli} get larger as it expresses
more ribosomes, an additional constraint begins to arise in energy production
due to a relative decrease in total surface area where ATP is predominantly
produced \citep{szenk2017}. Specifically, the cell interior requires an amount
of energy that scales cubically with cell size, but the available surface area
only grows quadratically (\FIG{energy_scaling}(A)). While this threshold does
not appear to be met for \textit{E. coli} cells growing at 2 hr$^{-1}$ or less,
it highlights an additional constraint on growth given the apparent need to
increase in cell size to grow faster. This is also potentially relevant to
eukaryotic organisms, whose mitochondria exhibit convoluted membrane structures
that nevertheless remain bacteria-sized organelles \citep{guo2018}. In the
context of bacteria growth and energy production more generally, we have limited
our analysis to the aerobic growth conditions associated with the proteomic data
and further consideration will be needed for anaerobic growth.

This work is by no means meant to be an exhaustive dissection of bacterial
growth rate control, and there are many aspects of the bacterial proteome and
growth that we neglected to consider. For example, other recent work
\citep{liebermeister2014, hui2015, schmidt2016} has explored how the proteome is
structured and how that structure depends on growth rate. In the work of
\cite{hui2015}, the authors coarse-grained the proteome into six discrete
categories being related to either translation, catabolism, anabolism, and
others related to signaling and core metabolism. The relative mass fraction of
the proteome occupied by each sector could be modulated by external application
of drugs or simply by changing the nutritional content of the medium. While we
have explored how the quantities of individual complexes are related to cell
growth, we acknowledge that higher-order interactions between groups of
complexes or metabolic networks at a systems-level may reveal additional
insights into how these growth-rate dependences  are mechanistically achieved.
Furthermore, while we anticipate the conclusions summarized here are applicable
to a wide collection of bacteria with similar lifestyles as \textit{E. coli},
other bacteria and archaea may have evolved other strategies that were not
considered. Further experiments with the level of rigor now possible in
\textit{E. coli} will need to be performed in a variety of microbial organisms
to learn more about how regulation of proteomic composition and  growth rate
control has evolved over the past 3.5 billion years.

%
%
%
%
% our estimates are
% appear to be in excess of what would be minimally required to support cell
% growth at the observed rates.
%
%
%  to elucidate a
% fundamental question in bacterial physiology -- what sets the speed limit at
% which cells can divide?


%
% Recent years have seen an explosion in our understanding of the cellular
% macromolecular composition at unprecedented resolution. This quantitative,
% molecular view of cell biology has transformed our understanding of how and when
% genes are expressed, to what degree they are expressed, and precisely how they
% are post-translationally modified (a topic we have not considered in this work,
% but appreciate nonetheless). Despite these impressive studies, an understanding
% of how the abundance and regulation is related to growth rate has been largely
% treated with phenomenological models, often containing obscure dimensionless
% parameters or polynomial fits to data. While these phenomenological treatments
% have revealed fascinating features of the resource allocation within the
% proteome, elucidating the molecular details of \textit{how} these resource
% allocation strategies operate has remained enigmatic.
%
% In this work, we present a series of order-of-magnitude estimates to elucidate a
% fundamental question in bacterial physiology -- what sets the speed limit at
% which cells can divide?
%
%
% Countless careers have been built examining the minutiae of each of
% the processes listed in \FIG{categories}, resulting in a treasure
% trove of literature representing decades of careful experimentation, modeling,
% and interpretation of results. Despite the undeniable complexity of each
% individual process, our work illustrates that in order to understand what sets
% the scale of the observed protein copy numbers and their corresponding
% dependence on growth rate, a coarse-grained view of the system is often
% appropriate.
%
%
% [GC: Other topics to discuss include the utility of scrounging the data from the
% literature to assemble, to our knowledge, the most complete view of the
% condition-dependent proteome of \textit{E. coli}.]
%
%
% [Fill in.]

% Parallelized DNA replication represents a solution that allows  \textit{E. coli}
% to synthesize enough rRNA as it grows faster. That the cell appears to
% be synthesizing rRNA near its maximal rate at growth rates above about 0.5 h$^{-1}$
% may also have consequences on robust scaling  between
% cell size and $\langle$\# ori$\rangle$. As one example, when cells are exposed
% to sublethal doses of ribosome-inhibiting drugs like chloramphenicol, there is a
% notable increase in the cell's ribosomal fraction (grey points,
% \FIG{translation_ecoli_partA}(D)), but otherwise the cell is able to maintain
% its cell size according to $\langle$\# ori$\rangle$ \citep{si2017}. While the
% presence of chloramphenicol will inhibit protein synthesis, it will also allow
% for relatively higher rRNA synthesis due to the longer doubling time. If the
% cell can scale it ribosomal protein abundance through feedback from rRNA
% synthesis, than we would expect the relative abundance of ribosomes to increase
% according to the increase in rRNA synthesis. This type of feedback was
% considered long ago \citep{nomura1984} and we consider this possibility further
% in Supplemental Appendix XX.


% While it is difficult to distinguish between causality and correlation, the data
% also helps us resolve why cell size, should exhibit an exponential increase with growth
% rate once cells begin to parallelize DNA replication. Specifically,
%
% is consistent with the need for cells to increase their effective rRNA gene
% dosage in order to grow according to the constraint set by Equation 2. Importantly, it
% may also shed some light on the notable increase in ribosomal content
% that is observed when sublethal doses of antibiotics \citep{scott2010, dai2016}.
% Specifically, if rRNA synthesis is rate limiting, and nutrient conditions
% largely dictate the extent of overlapping DNA replication cycles, than addition
% of antibiotic will lengthen the doubling time and allow increased rRNA
% synthesis relative to the rate of cell division. In Supplemental Section XX, we
% consider this further using additional data from \cite{si2017}.


% A number of recent papers further highlight the possibility that regulation
% (p)ppGpp may be the critical component of the apparent scaling laws in
% \textit{E. coli}. In the context of ribosomal activity, increased levels of
% (p)ppGpp are associated with lower ribsomal content, and at slow growth
% are associated with reduced activity by ribosomes and RNA polymerse \citep{dai2016,
% dai2018} [citations for RNAP]. Titration of the cellular (p)ppGpp concetrations (up or down) can
% invoke similar proteomic changes reminiscent of those observed under nutrient
% limitation \citep{zhu2019}. In light of the limiting dependence of ribosome copy
% number on chromosomal gene dosage, it was recently shown that growth in a
% (p)ppGpp null strain abolishes both the scaling in cell size  and the
% $\langle$\# ori$\rangle$ / $\langle$\# ter$\rangle$ ratio. Instead, cells
% exhibited a high $\langle$\# ori$\rangle$ / $\langle$\# ter$\rangle$ closer to 4
% and cell size more consistent with a fast growth state where (p)ppGpp levels are
% low \citep{fernandezcoll2020}. From these results, perhaps the perspective to
% take is that the scaling laws  reflect an attempt for the cell to mitigate its
% biological activity according  to available nutrients.
