\section{Function of the Central Dogma}
Up to this point, we have considered a variety of transport and biosynthetic
processes that are critical to acquiring and generating new cell mass. While
there are of course many other metabolic processes we could consider and
perform estimates of (such as the components of fermentative versus aerobic
respiration), we now turn our focus to some of the most central processes
which \textit{must} be undertaken irrespective of the growth conditions --
the processes of the central dogma.

\subsection{DNA Replication}
Most bacteria, like \textit{E. coli}, harbor a single,
circular chromosome and can have extra-chromosomal plasmids $\sim$ 100 kbp in
length. We will focus our quantitative thinking solely on the chromosome of
\textit{E. coli} which harbors $\approx$ 5000 genes and $\approx 5\times
10^6$ base pairs. To successfully divide and produce viable progeny, this chromosome must be faithfully replicated and
segregated into each nascent cell. We again rely on the
near century of literature in molecular biology to provide some insight
towards the rates and mechanics of the replicative feat. Replication of the
bacterial chromosome is initiated at a single region of the chromosome termed
the \textit{oriC} locus at which a pair of DNA polymerases bind and begin
their high-fidelity replication of the genome in opposite directions.
Assuming equivalence between the two replication forks, this means that the
two DNA polymerases meet at the midway point of the circular chromosome
termed the \textit{ter} locus. This divison of labor means The kinetics of
the four types of DNA polymerases (I -- V) have been intensely studied,
revealing that DNA polymerase III performs the high fidelity processive
replication of the genome with the other "accessory" polymerases playing
auxiliary roles \cite{fijalkowska2012}. \textit{In vitro} measurements have
shown that DNA Polymerase III copies DNA at a rate of $\approx 600$
nucleotides per second (BNID: 104120, \cite{milo2010}). Thus, to replicate a
single chromosome, two DNA polymerases replicating at their maximal rate would
copy the entire genome in $\approx$ 4000 s. Thus, with a division time of 6000
s (our "typical" growth rate for the purposes of this work), there is sufficient
time for a pair of DNA polymerases to replicate the entire genome.  However, this
estimate implies that 4000 s would be the upper-limit time scale for bacterial
division which is at odds with the familiar $\approx$ 1500 s doubling time of
\textit{E. coli} in rich medium.

It is know well known that \textit{E. coli} can parallelize its DNA replication
such that multiple chromosomes are being replicated at once. Recent work
\citep{si2017} has shown that the replicative timescale of cell division can be
massively parallelized where \textit{E. coli} can have on the order of 10 - 12
replication forks at a given time.  Thus, even in rapidly growing cultures, only
a few polymerases ($\approx 10$) are needed to replicate the chromosome.
However, as shown in \FIG{dna_synthesis}(A), DNA polymerase III is nearly an 
nearly an order of magnitude more abundant. This discrepancy can be understood
when considering the binding affinities  [GC: ... need to think this out. This
feels like I am going far afield of the actual message of the paper.]

\subsection{dNTP Synthesis}